<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UISettings</class>
 <widget class="QDialog" name="UISettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <widget class="QListWidget" name="lwPages">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>General</string>
      </property>
      <property name="icon">
       <iconset resource="resources/resources.qrc">
        <normaloff>:/icons/find-options-16.png</normaloff>:/icons/find-options-16.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Roms</string>
      </property>
      <property name="icon">
       <iconset resource="resources/resources.qrc">
        <normaloff>:/icons/find-clear-16.png</normaloff>:/icons/find-clear-16.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Performance</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Video</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Misc</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lInformations">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>General</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QStackedWidget" name="swPages">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="wGeneral">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="topMargin">
          <number>12</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="lBinaryFile">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Binary file</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="Line" name="fBinaryFile">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="leBinaryFile">
           <property name="toolTip">
            <string>Enter the 'sdlmess' binary filepath/filename</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="tbBinaryFile">
           <property name="toolTip">
            <string>Locate 'sdlmess' binary file</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="topMargin">
          <number>12</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="lBiosPath">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Bios path</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="Line" name="fBiosPath">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="leBiosPath">
           <property name="toolTip">
            <string>Enter the full path of your bios</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="tbBiosPath">
           <property name="toolTip">
            <string>Locate your bios path</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>92</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wRoms">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="topMargin">
          <number>12</number>
         </property>
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="lRomsPath">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Roms path</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="Line" name="fRomsPath">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="leRomsPath">
           <property name="toolTip">
            <string>Enter a roms path for this machine</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QToolButton" name="tbRomsPath">
           <property name="toolTip">
            <string>Locate a roms path</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QComboBox" name="cbRomsPath">
           <property name="toolTip">
            <string>List all non clone machines</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QToolButton" name="tbRomsPathSet">
           <property name="toolTip">
            <string>Validate the current roms path</string>
           </property>
           <property name="icon">
            <iconset resource="resources/resources.qrc">
             <normaloff>:/icons/find-options-16.png</normaloff>:/icons/find-options-16.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>132</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wPerformance">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="cbMultithreading">
         <property name="text">
          <string>Activate multithreading</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbSDLFPS">
         <property name="text">
          <string>Show SDL FPS</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>173</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wVideo">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lVideoMode">
           <property name="text">
            <string>Video mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbVideoMode">
           <item>
            <property name="text">
             <string>soft</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>opengl</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lScaleMode">
           <property name="text">
            <string>Scale mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbScaleMode">
           <item>
            <property name="text">
             <string>none</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>async</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>yv12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>yuy2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>yv12x2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>yuy2x2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cbFullScreen">
         <property name="text">
          <string>FullScreen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbMaximize">
         <property name="text">
          <string>Maximize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbKeepAspectRatio">
         <property name="text">
          <string>Keep aspect ratio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbAllowDoubleStretchFactors">
         <property name="text">
          <string>Allow non-integer stretch factors</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbWaitVBLANK">
         <property name="text">
          <string>Wait for VBLANK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>32</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="cbSkipGameInformations">
         <property name="text">
          <string>Skip game informations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbSkipWarnings">
         <property name="text">
          <string>Skip warnings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbCheat">
         <property name="text">
          <string>Activate cheat engine</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>146</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="dbbButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lwPages</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>lInformations</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lwPages</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>swPages</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbbButtons</sender>
   <signal>accepted()</signal>
   <receiver>UISettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbbButtons</sender>
   <signal>rejected()</signal>
   <receiver>UISettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clicked(QAbstractButton*)</slot>
 </slots>
</ui>
